firestore-genai-chatbot/functions/src/generative-client/google_ai.ts

```diff
@@ -24,103 +25,57 @@ type ApiMessage = {
   }[];
 };
 
-enum Role {
-  USER = 'user',
-  GEMINI = 'model',
-}
+const ai = genkit({
+  plugins: [googleAI()],
```

Comment:
need to make sure api key is passed in here correctly, the plugin wont work without it